Gemfile

```diff
@@ -1,24 +1,25 @@
 source 'https://rubygems.org'
 
 gem 'sinatra', '~> 4.0.0'
-gem 'nokogiri', '~> 1.16'
-gem 'rake', '~> 13.0.0'
-gem 'puma'
 gem 'prometheus-client'
-gem 'rack', '~> 3.0.0'
+gem 'puma'
 gem 'newrelic_rpm'
+gem 'nokogiri', '~> 1.16'
+gem 'rack', '~> 3.0.0'
+gem 'rake', '~> 13.0.0'
 
 group :test do
   gem 'capybara'
   gem 'capybara-selenium'
   gem 'pry-byebug'
+  gem 'rackup'
   gem 'rack-test', '>= 2.0.0'
   gem 'rspec'
 end
 
 group :development, :test do
-  gem 'aamva', github: '18F/identity-aamva-api-client-gem', tag: 'v4.1.0'
-  gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.4.1'
-  gem 'lexisnexis', github: '18F/identity-lexisnexis-api-client-gem', tag: 'v3.1.0'
+  gem 'aamva', github: '18F/identity-aamva-api-client-gem', tag: 'v4.2.0'
+  gem 'identity-doc-auth', github: '18F/identity-doc-auth', tag: 'v0.13.0'
+  gem 'lexisnexis', github: '18F/identity-lexisnexis-api-client-gem', tag: 'v3.2.1'
```

Comment:
I'd just remove these entirely since we're no longer using them

Reply:
Will do.